content/en/docs/tasks/network/extend-service-ip-ranges.md

```diff
@@ -0,0 +1,182 @@
+---
+reviewers:
+- thockin
+- dwinship
+min-kubernetes-server-version: v1.29
+title: Extend Service IP Ranges
+content_type: task
+---
+
+<!-- overview -->
+{{< feature-state state="alpha" for_k8s_version="v1.29" >}}
+
+This document shares how to extend the existing Service IP range assigned to a cluster.
+
+
+## {{% heading "prerequisites" %}}
+
+{{< include "task-tutorial-prereqs.md" >}}
+
+{{< version-check >}}
+
+<!-- steps -->
+
+## API
+
+Kubernetes clusters that are created with the `MultiCIDRServiceAllocator`
```

Comment:
Created? Can you turn this on after cluster creation?

Reply:
it is an option in the apiserver, so I will rephrase to make this more explicit,